### DISCUSSION TITLE
MSCS%40UWM/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#122] [Announcements]
# MSCS%40UWM/

https://csmsapp.org/MSCS%40UWM/

**Comment by ZhuoruLi** [comment]
等等nb姐来更新

**Comment by yuhengcai1** [comment]
并不是陆本黑洞，只是这项目一般只录取陆本年纪第一，因为是全奖。而且MSCS里面陆本出身的人应该不少，而且这个项目应该算是比较陆本友好的，相比UIUC和Cornell的全被自家人包围的情况，这个项目基本上都收外校学生

**Reply by Met4physics** [reply]
官网写的不是只有一半左右的人有funding吗？这个项目难度如何，计算机弱校211第一有机会吗，有没有funding无所谓。听说挺看重pub的。求大佬解答

**Reply by xichenpan** [reply]
> 官网写的不是只有一半左右的人有funding吗？这个项目难度如何，计算机弱校211第一有机会吗，有没有funding无所谓。听说挺看重pub的。求大佬解答

@yuhengcai1 帮你at他一下hh

**Reply by Met4physics** [reply]
哈哈，谢谢谢谢

**Reply by yuhengcai1** [reply]
被信件吵起来了，首先这个项目是百分百的全奖，但是你有没有机会我不知道，我只能说你试试吧，确实很看pub，毕竟是一个研究型项目，但是整体看下来陆本挺多的，不过认识的唯一一个是浙大的专业课第一，所以我只能说这玩意有难度

**Reply by Met4physics** [reply]
好的好的非常感谢
